GoedeDoelenHelpen/ClientApp/src/app/user/dashboard/live-comments/live-comments.component.html

```diff
@@ -0,0 +1,13 @@
+<mat-card class="partial" id="live_comments">
+  <ul>
+    <li>
+      Live comment 1
+    </li>
+    <li>
+      Live comment 2
+    </li>
+    <li>
+      Live comment 3
```

Comment:
Waar de dropdown bij de header netjes uit een array wordt gehaald, staat dit er hardcoded in.
Incosistent, maar geen groot probleem.